adobe/spectrum-web-components
# fix: ensure icon packages get properly processed for custom-elements.json listings

## Description
Update the cem config so it doesn't exclude the icons files.
Update the checking scripts to include these packages.

## Related issue(s)
- fixes #2773

## How has this been tested?
    1. Pull down this branch.
    2. Run `yarn`
    3. Run `yarn custom-element-json`
    4. See that `packages/icons-workflow/custom-elements.json` lists lots of files and is almost 40K lines long.

## Types of changes

## Checklist
